docs/HowToUpgrade.md

```diff
@@ -199,31 +220,28 @@ If the rollout doesn't progress for a long time, refer to the NuoDB pods using `
 Watch for `Running` pods in which containers are _not_ `Ready`, which usually means that there is a problem with the application startup.
 Another indication for upgrade failure is having pods _not_ transiting to `Running` or reported `Error`, `ErrImagePull` or `CrashLoopBackOff`.
```

Comment:
This should read, "Another indication of upgrade failure ..." (that is, replace "for" with "of")